### PR TITLE
Ensure Withdrawn is included on all withdrawn content

### PR DESCRIPTION
**Pull request by dwhenry**
This cover a few edge cases where custom page_titles
where being generated

https://trello.com/c/Mn8Yk9Z6/278-add-withdrawn-to-page-title

### CODE REVIEW

File: test/presenters/guide_presenter_test.rb [hunk 1]
```diff
@@ -17,9 +17,9 @@ def schema_name
       end
     end
 
-    test "presents part titles with their index" do
```
Comment (fofr) [inline test/presenters/guide_presenter_test.rb]: Shouldn't the old test remain?

Comment (dwhenry) [reply test/presenters/guide_presenter_test.rb]: umm... yes, this has been reverted..
